ui/src/app/base/components/DhcpSnippetType/DhcpType.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { render, screen } from "@testing-library/react";
+import { Provider } from "react-redux";
+import configureStore from "redux-mock-store";
+
+import DhcpType from "./DhcpSnippetType";
```

Comment:
Is the different filename here an accident?

Reply:
Good spot - missed this when renaming. Updated.